framework/include/tinyalsa/tinyalsa.h

```diff
@@ -262,6 +262,24 @@ struct pcm;
 struct pcm *pcm_open(unsigned int card, unsigned int device, unsigned int flags, const struct pcm_config *config);
 
 /**
+* @brief Stops a PCM. Any data present in the buffers will be dropped.
+*
+* @param[in] A PCM handle.
```

Comment:
1. You should write a name of param.
```
 * @param[in] pcm A PCM handle.
```
2. Please add details for including method.
```
 * @details @b #include <tinyalsa/tinyalsa.h>
```

Reply:
Done